lib/context.js

```diff
@@ -73,19 +73,20 @@ const proto = module.exports = {
   assert: httpAssert,
 
   /**
-   * Throw an error with `msg` and optional `status`
-   * defaulting to 500. Note that these are user-level
+   * Throw an error with `status` (default 500) and
+   * `msg`. Note that these are user-level
    * errors, and the message may be exposed to the client.
    *
    *    this.throw(403)
-   *    this.throw('name required', 400)
    *    this.throw(400, 'name required')
    *    this.throw('something exploded')
-   *    this.throw(new Error('invalid'), 400);
-   *    this.throw(400, new Error('invalid'));
+   *    this.throw(new Error('invalid'))
+   *    this.throw(400, new Error('invalid'))
    *
    * See: https://github.com/jshttp/http-errors
    *
+   * Note: `stats` should only be passed as the first parameter.
```

Comment:
This was misspelled, fixed in https://github.com/koajs/koa/pull/1269

Reply:
Oh, thanks 🎉